### ISSUE TITLE
Can not set hyperparameters during tuning

### ISSUE THREAD

**Issue by nlm-yuh5** [ultralytics/ultralytics#5127]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi there,

I was trying to set 'lrf=1.0' and 'workers=2' during tuning, however, the default values still remained after tuning started. Please help. Below is my code:

`model.tune(data=data_yaml, epochs=epoch, iterations=iterations, optimizer=optimizer_str, plots=False, save=False,
               val=False, batch=-1, imgsz=1024, lrf=1.0, dropout=0.5, rect=True, project=out_dst+str(folder_num)+'/',
               workers=2)`

**Comment by glenn-jocher** [comment]
@nlm-yuh5 hello and thank you for reaching out with your issue.

Sometimes, issues like this can be caused by an incorrect use of arguments or an erroneous understanding of the method's functionality. The hyperparameters you're trying to set ('lrf' and 'workers') might not be affecting the model tuning due to such reasons.

The 'lrf' argument in the `tune()` function adjusts the learning rate finder, whereas 'workers' is the number of subprocesses for data loading. These hyperparameters could be limited due to your system constraints or specific settings in your codebase.

Regarding 'workers', make sure your system has enough resources to execute multiple processes. If not, the 'workers' value may be reset to a default.

On the other hand, 'lrf' does not directly set the learning rate but rather influences the learning rate finder's functioning.

I suggest double-checking the hyperparameter handling in your setting and considering other factors that might be influencing their usage.

If you have more questions, don't hesitate to ask. I'm here to assist you!
